charts/crljanitor/values.yaml

```diff
@@ -41,6 +44,9 @@ serviceAllowedAddresses: {}
 
 # trackResourcePubsubEnabled -- Whether to enable using pubsub to receive new tracked resources.
 trackResourcePubsubEnabled: true
+# configBasedAuthZEnabled -- Whether to use static file with admin users email for authorization.
+# TODO(PF-81): Switch to use SAM instead of config file for user authZ.
+configBasedAuthZEnabled: true
```

Comment:
super naming nit (feel free to punt if you disagree): the env variable treats "authz" as one token, but the capitalization here treats "z" as its own token. IMO it's more consistent to keep "authz" as one token, which would make this config name "configbasedAuthzEnabled" (also in the corresponding PR).

Reply:
Done. Thanks!